app/models/user.rb

```diff
@@ -1,9 +1,11 @@
-class User < ActiveRecord::Base
+class User < ApplicationRecord
+  validates :name, presence: true
+
```

Comment:
- [ ] As requested in the project requirements, kindly add validation for the post counter in the user model